webviews/components/history_panel.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Should this be committed here?
Is it just for testing the transition?
Also, this shouldn't be a separate HTML file, just a different view in the same svelte app.